utils/hooks/gitblit-only/verigreen.groovy

```diff
@@ -64,6 +69,8 @@ private def executeOnShell(String command, File workingDir, StringBuilder output
     if (!JAVA_HOME.equals("")) {     // use the JAVA_HOME if it is defined (earlier in this script)
         env.put("JAVA_HOME", JAVA_HOME)
     }
+	logger.info("VG_HOOK:" + VG_HOOK + ln ) 
```

Comment:
Not all messages are needed for production, I'll get to adding a variable to control this

Eli

From: eitanshp [mailto:notifications@github.com]
Sent: Wednesday, August 26, 2015 2:04 PM
To: Verigreen/verigreen <verigreen@noreply.github.com>
Cc: Dagan, Eliyahu <dagan@hp.com>
Subject: Re: [verigreen] Gitblit groovy script updates: (#51)


In utils/hooks/gitblit-only/verigreen.groovy<https://github.com/Verigreen/verigreen/pull/51#discussion_r37969574>:

> @@ -64,6 +69,8 @@ private def executeOnShell(String command, File workingDir, StringBuilder output

>      if (!JAVA_HOME.equals("")) {     // use the JAVA_HOME if it is defined (earlier in this script)

>          env.put("JAVA_HOME", JAVA_HOME)

>      }

> +     logger.info("VG_HOOK:" + VG_HOOK + ln )

Do you need those for production?

—
Reply to this email directly or view it on GitHub<https://github.com/Verigreen/verigreen/pull/51/files#r37969574>.